### ISSUE TITLE
Consistency in underscores vs. hyphens for identifiers

### ISSUE THREAD

**Issue by mrwojo** [gravitystorm/openstreetmap-carto#2072]
It's a minor annoyance that some identifiers use underscores whereas others use hyphens, like "leisure_nature_reserve" and "nature-reserve-boundaries".

Could/should there be more consistency?

**Comment by math1985** [comment]
Yes, we definitely should aim for consistency! I would welcome a pull request to fix this.

**Comment by Adamant36** [comment]
If I wanted to work on this, how would I go about it? Just turn all the hyphens in the amenity-points.mss into underscores or visa versa? Also, which character is preferred?

**Comment by kocio-pl** [comment]
I'm not sure which is better, but it means testing if things don't break, so maybe it should be cut into smaller, well-defined changes.
